crates/e2e/src/setup/deploy.rs

```diff
@@ -185,6 +185,31 @@ impl Contracts {
         }
     }
 
+    // Delete when flashloan is actually deployed
+    pub async fn deploy_flashloan(self: Self, web3: &Web3) -> Self {
+        macro_rules! deploy {
+                ($contract:ident) => { deploy!($contract ()) };
+                ($contract:ident ( $($param:expr),* $(,)? )) => {
+                    deploy!($contract ($($param),*) as stringify!($contract))
+                };
+                ($contract:ident ( $($param:expr),* $(,)? ) as $name:expr) => {{
+                    let name = $name;
+                    $contract::builder(&web3 $(, $param)*)
+                        .deploy()
+                        .await
+                        .unwrap_or_else(|e| panic!("failed to deploy {name}: {e:?}"))
+                }};
+            }
+
+        let flashloan_wrapper =
+            deploy!(ERC3156FlashLoanSolverWrapper(self.gp_settlement.address()));
```

Comment:
Nit: the `deploy!` macro is useful when there are many contracts to deploy but since we only call it once here it literally just adds confusion.

Reply:
resolved